prac_06/programming_language.py

```diff
@@ -0,0 +1,17 @@
+
+
+class ProgrammingLanguage:
+
+    def __init__(self, name, typing, reflection, year):
+        """initialise a programming language instance """
+        self.name = name
+        self.typing = typing
+        self.reflection = reflection
+        self.year = year
+
+    def is_dynamic(self):
+        """check if language is dynamic"""
+        return self.typing == "Dynamic"
+
+    def __str__(self):
```

Comment:
Maybe move this above is_dynamic() so that the overridden "\_\_" methods are together